File: app/src/main/java/com/github/batulovandrey/unofficialurbandictionary/view/FavoritesView.java
package com.github.batulovandrey.unofficialurbandictionary.view;

/**
 * @author Andrey Batulov on 30/10/2017
 */

public interface FavoritesView {

    void showAlertDialog();

    void showToast(int resId);

    void hideRecycler();

    void showRecycler();
}

File: app/src/main/java/com/github/batulovandrey/unofficialurbandictionary/view/FavoritesView.kt
package com.github.batulovandrey.unofficialurbandictionary.view

/**
 * @author Andrey Batulov on 22/12/2017
 */

interface FavoritesView {

    fun showAlertDialog()

    fun showToast(resId: Int)

    fun hideRecycler()

    fun showRecycler()
}